apps/web/src/containers/TradingContest/components/contest-countdown/contest-countdown.tsx

```diff
@@ -1,78 +1,53 @@
-import { Box, styled } from '@mui/material';
+import { Box, SxProps, styled, useTheme } from '@mui/material';
 import { colors } from '@notional-finance/styles';
-import { FormattedMessage } from 'react-intl';
 import Countdown from 'react-countdown';
-import { Button } from '@notional-finance/mui';
+import { SectionTitle } from '@notional-finance/mui';
 import { ReactNode } from 'react';
 
 interface ContestCountDownProps {
-  title: ReactNode;
+  title?: ReactNode;
+  sx?: SxProps;
 }
 
-export const ContestCountDown = ({ title }: ContestCountDownProps) => {
-  // This is a millisecond timestamp for 12 AM on 10/25/2023
-  const futureDate = 1698210000000;
+export const ContestCountDown = ({ title, sx }: ContestCountDownProps) => {
+  const theme = useTheme();
+  // This is a millisecond timestamp for 12 AM on 02/01/2024
+  const futureDate = 1706832000000;
```

Comment:
Can you put this constant in a contest config file or something so we can update it more easily?